frontend/README.md

```diff
@@ -0,0 +1,16 @@
+made with next.js
```

Comment:
**question**: Why did you choose NextJS over React? I know that NextJS is a superset of React, just curious about your choice.